content/tokio/tutorial/hello-tokio.md

```diff
@@ -94,155 +157,58 @@ The [`client::connect`] function is provided by the `mini-redis` crate. It
-
-[`async/await`]: https://en.wikipedia.org/wiki/Async/await
-
-If this doesn't quite make sense yet, don't worry. We will explore `async/await`
-more throughout the guide.
-
-## Using `async/await`
-
-Async functions are called like any other Rust function. However, calling these
-functions does not result in the function body executing. Instead, calling an
-`async fn` returns a value representing the operation. This is conceptually
-analogous to a zero-argument closure. To actually run the operation, you should
-use the `.await` operator on the return value.
-
-For example, the given program
-
-```rust
-async fn say_world() {
-    println!("world");
-}
-
-#[tokio::main]
-async fn main() {
-    // Calling `say_world()` does not execute the body of `say_world()`.
-    let op = say_world();
-
-    // This println! comes first
-    println!("hello");
-
-    // Calling `.await` on `op` starts executing `say_world`.
-    op.await;
-}
-```
-
-outputs:
-
-```text
-hello
-world
+# use mini_redis::client;
+# async fn dox() -> mini_redis::Result<()> {
+# let mut client = client::connect("127.0.0.1:6379").await?;
+client.set("hello", "world".into()).await?;
+# Ok(())
+# }
 ```
 
-The return value of an `async fn` is an anonymous type that implements the
-[`Future`] trait.
+To get the value of a key, we use the [`Client::get`] method, which also requires `await`.
 
-[`Future`]: https://doc.rust-lang.org/std/future/trait.Future.html
-
-## Async `main` function
-
-The main function used to launch the application differs from the usual one
-found in most of Rust's crates.
-
-1. It is an `async fn`
-2. It is annotated with `#[tokio::main]`
-
-An `async fn` is used as we want to enter an asynchronous context. However,
-asynchronous functions must be executed by a [runtime]. The runtime contains the
-asynchronous task scheduler, provides evented I/O, timers, etc. The runtime does
-not automatically start, so the main function needs to start it.
-
-The `#[tokio::main]` function is a macro. It transforms the `async fn main()`
-into a synchronous `fn main()` that initializes a runtime instance and executes
-the async main function.
-
-For example, the following:
+[`Client::get`]: https://docs.rs/mini-redis/latest/mini_redis/client/struct.Client.html#method.get
 
 ```rust
-#[tokio::main]
-async fn main() {
-    println!("hello");
-}
+# use mini_redis::client;
+# async fn dox() -> mini_redis::Result<()> {
+# let mut client = client::connect("127.0.0.1:6379").await?;
+let result = client.get("hello").await?;
+# Ok(())
+# }
 ```
 
-gets transformed into:
+The result is an `Option<Bytes>`, which will be `Some(Bytes)` if the key exists
+or `None` if it does not. We print the Debug format of the result.
 
-```rust
-fn main() {
-    let mut rt = tokio::runtime::Runtime::new().unwrap();
-    rt.block_on(async {
-        println!("hello");
-    })
-}
+```rust,ignore
+# let result = 42;
+println!("got value from the server; result={result:?}");
 ```
```

Comment:
This `let result = 42` line shows up in the rendered html for me. Perhaps you need to drop the `ignore` from the codeblock to fix this?